### PR TITLE
Drop 'please' from error messages

### PR DESCRIPTION
**Pull request by frankieroberto**
The [GOV.UK Design System guidance for error messages](https://design-system.service.gov.uk/components/error-message/) is:

> Be clear and concise
> Do not use ‘please’ because it implies a choice

### CODE REVIEW

**Review by peteryates — APPROVED** [review]

**Comment by peteryates** [comment]
Sorry approved before realising it was still draft 🙈

**Comment by frankieroberto** [comment]
> Sorry approved before realising it was still draft 🙈

Was just waiting for the tests to pass (and I missed one 🤦)

File: app/forms/appropriate_body_selection_form.rb [hunk 1]
```diff
@@ -15,14 +15,14 @@ class AppropriateBodySelectionForm
 
   validates :body_appointed,
             inclusion: { in: %w[yes no],
-                         message: "Please select whether you have appointed an appropriate body or not" },
+                         message: "Select whether you have appointed an appropriate body or not" },
```
Comment (Lloyd-K) [inline app/forms/appropriate_body_selection_form.rb:18]:
```suggestion
                         message: "Select whether you’ve appointed an appropriate body or not" },
```

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ en:
       failure: "Could not authenticate you from %{kind} because \"%{reason}\"."
       success: "Successfully authenticated from %{kind} account."
     passwords:
-      no_token: "You can’t access this page without coming from a password reset email. If you do come from a password reset email, please make sure you used the full URL provided."
+      no_token: "You can’t access this page without coming from a password reset email. If you do come from a password reset email, make sure you used the full URL provided."
```
Comment (Lloyd-K) [inline config/locales/devise.en.yml:34]:
```suggestion
      no_token: "You cannot access this page unless you received a password reset email. If you did receive a password reset email, make sure you used the full URL provided."
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -193,9 +193,9 @@ en:
       blank: "Not registered"
       not_active: "Not active"
     programme:
-      blank: "Please select programme type"
+      blank: "Select programme type"
```
Comment (Lloyd-K) [inline config/locales/en.yml:196]:
```suggestion
      blank: "Select a programme type"
```

**Review by Lloyd-K — CHANGES_REQUESTED** [review]

File: app/forms/appropriate_body_selection_form.rb (hunk 1 above)
Comment (Lloyd-K) [reply app/forms/appropriate_body_selection_form.rb:18]: Thanks for removing the 'please' instances. I've added extra amendments to bring all content in-line with the GDS style guide.

**Pull request merged**